db/sqlglue.c

```diff
@@ -574,6 +574,18 @@ static inline int check_recover_deadlock(struct sqlclntstate *clnt)
     return rc < 0 ? SQLITE_BUSY : rc;
 }
 
+static int isPrepare(BtCursor *pCur)
+{
```

Comment:
Given what this function actually checks, the name seems a bit confusing.  Perhaps it can be named something like "cursor_in_db_init()" ?